app/schema.py

```diff
@@ -114,7 +114,6 @@ class Config:
 class Meeting(BaseModel):
     meeting_id: constr(max_length=255)
     created_at: datetime
```

Comment:
Eventually I'd like to refactor all timestamping to the database layer. This will unify how we handle created_at and updated_at such that we no long need them in the schema at all. This is great for now though!